jaws-journey-deploy/orb.yml

```diff
@@ -50,6 +52,13 @@ aliases:
         default: false
       <<: *environment
 
+  tf-command-parameters: &tf-command-parameters
+    path: << parameters.path >>
```

Comment:
why is this path << parameters.path >>, but tf-parameters path is terraform/<< parameters.path >>

Reply:
Just passing it through.

Reply:
yeah just wondering why we want them to be different I guess